### ISSUE TITLE
Feature Request: Save motion-recorded clip to SD 

### ISSUE THREAD

**Issue by Tchekda** [JurajNyiri/HomeAssistant-Tapo-Control#90]
**Is your feature request related to a problem? Please describe.**
In the App, there is a toggle to enable/disable the saving of clips to the SD card when the motion sensors is enabled. But this toggle resets to No when I disable the motion sensor with HA

**Describe the solution you'd like**
Either an action to enable the recording save. Or add it as an option to the service that enables the motion sensor.

**Describe alternatives you've considered**
For now, I have to open the app and toggle manually, so I'm loosing the time that HA is supposed to do for me.

**Additional context**
If you need any help for testing/debugging, I will be more than happy to help you.

**Comment by JurajNyiri** [comment]
Is this the same with the app? What happens when you disable motion sensor in the app?

**Comment by Tchekda** [comment]
> Is this the same with the app? What happens when you disable motion sensor in the app? 

I don't know what happened with my camera but when I opened this issue the toggle was being reset each time I was making an HA call.

I rebooted the camera for another reason and it's working as expected since. So I don't have  to manually trigger the "save to SD" toggle each time.

I'm so sorry for bothering, I should have applied the first rule of technology : "Have you tried turning it off and then on ?".

Closing this issue
